client/src/components/Product.jsx

```diff
@@ -46,33 +44,34 @@ class Product extends React.Component {
       answered_questions: exampleData.answered_questions,
     };
     // This changes the title of the page (title in the browser tab) to:
-    // Amazon.com: Whatever product name
+    // Amazon.com: Product name
     document.title = `Amazon.com: ${exampleData.name}`;
   }
 
   render() {
     return (
       <App>
-        <div className="pictures-section red-border">
+
+        <PicturesSection>
           <PicturesView pictureURLs={this.state.image_urls} />
-        </div>
-        <div>
-          <div style={{display: 'inline-block'}}>
-            <TitleView title={this.state.name} seller={this.state.seller} />
-            <ReviewsView
-              one={this.state.one_star_reviews}
-              two={this.state.two_star_reviews}
-              three={this.state.three_star_reviews}
-              four={this.state.four_star_reviews}
-              five={this.state.five_star_reviews}
-              total={this.state.total_reviews}
-              answered_questions={this.state.answered_questions} />
-            <div className="line"></div>
-          </div>
-          <PriceView price={this.state.versions.new.price} />
-          <DescriptionView description={this.state.description} />
-          <div className="line"></div>
-        </div>
+        </PicturesSection>
+
+        <ContentSection>
+          <TitleView title={this.state.name} seller={this.state.seller} />
+          <ReviewsView
+            one={this.state.one_star_reviews}
+            two={this.state.two_star_reviews}
+            three={this.state.three_star_reviews}
+            four={this.state.four_star_reviews}
+            five={this.state.five_star_reviews}
+            total={this.state.total_reviews}
+            answered_questions={this.state.answered_questions} />
```

Comment:
A very small stylistic choice. Do you think it would be appropriate to move the /> to a new line?

Reply:
I don't really know. I thought the same thing myself when writing this line of code and decided not to for no particular reason.